include/boost/heap/detail/heap_comparison.hpp

```diff
@@ -26,6 +26,7 @@ namespace boost { namespace heap { namespace detail {
 template < typename Heap1, typename Heap2 >
 bool value_equality( Heap1 const& lhs, Heap2 const& rhs, typename Heap1::value_type lval, typename Heap2::value_type rval )
 {
+	[[maybe_unused]] rhs;
```

Comment:
maybe_unused is c++17, this library is still c++14. `boost::ignore_unused` would be more compatible.

also, please use spaces instead of tabs (or ideally use pre-commit to format the code